VALOAM:

  #mode
  debugging: false
  mode: true #true: fusion, false: origin

  # Topics
  odometryTopic: "/odom"
  pointCloudTopic: "/points_raw"               # Point cloud data
  lidarEdgeTopic: "/laser_cloud_edge"
  lidarSurfTopic: "/laser_cloud_surf"
  imageTopic: "/kitti/camera_gray_left/image_raw"               # Point cloud data
  # sensor info
  scan_period: 0.1

  #result out
  resultPath: "/home/taeki/catkin_ws/src/RESULT/6dof/ISC/fusion/loop/a.txt"
  
  # Extrinsics: T_lb (lidar -> imu)
  #0926 #0930 #1003
  extrinsicTrans: [-8.086759e-01, 3.195559e-01,-7.997231e-01]
  extrinsicRot: [ 9.999976e-01, 7.553071e-04, -2.035826e-03,
                 -7.854027e-04, 9.998898e-01, -1.482298e-02 ,
                  2.024406e-03 ,1.482454e-02 ,9.998881e-01]
  extrinsicRPY: [ 9.999976e-01, 7.553071e-04, -2.035826e-03,
                 -7.854027e-04, 9.998898e-01, -1.482298e-02 ,
                  2.024406e-03 ,1.482454e-02 ,9.998881e-01]

  #Extrinsics: T_lc (lidar -> camera)
  # 2011_09_30_drive_0018_sync
  # Tr_velo_to_camV: [ 7.027555e-03, -9.999753e-01, 2.599616e-05, -7.137748e-03,
  #                  -2.254837e-03, -4.184312e-05, -9.999975e-01, -7.482656e-02,
  #                   9.999728e-01, 7.027479e-03, -2.255075e-03, -3.336324e-01]
  # R0_rectV:        [ 9.999280e-01, 8.085985e-03, -8.866797e-03, 
  #                  -8.123205e-03, 9.999583e-01, -4.169750e-03,
  #                   8.832711e-03, 4.241477e-03, 9.999520e-01]
  # P0V:             [ 7.070912e+02, 0.000000e+00, 6.018873e+02, 0.000000e+00,
  #                   0.000000e+00, 7.070912e+02, 1.831104e+02, 0.000000e+00,
  #                   0.000000e+00, 0.000000e+00, 1.000000e+00, 0.000000e+00]
  
  #2011_09_26_drive_0051_sync 
  #2011_09_26_drive_0101_sync
  # Tr_velo_to_camV: [ 7.533745e-03, -9.999714e-01, -6.166020e-04, -4.069766e-03,
  #                   1.480249e-02, 7.280733e-04, -9.998902e-01, -7.631618e-02,
  #                   9.998621e-01, 7.523790e-03, 1.480755e-02, -2.717806e-01]
  # R0_rectV:        [ 9.999239e-01, 9.837760e-03, -7.445048e-03, 
  #                  -9.869795e-03, 9.999421e-01, -4.278459e-03,
  #                   7.402527e-03, 4.351614e-03, 9.999631e-01]
  # P0V:             [ 7.215377e+02, 0.000000e+00, 6.095593e+02, 0.000000e+00,
  #                   0.000000e+00, 7.215377e+02, 1.728540e+02, 0.000000e+00,
  #                   0.000000e+00, 0.000000e+00, 1.000000e+00, 0.000000e+00]
  #2011_10_03_drive_0042_sync 
  #2011_10_03_drive_0047_sync
  Tr_velo_to_camV: [ 7.967514e-03, -9.999679e-01, -8.462264e-04, -1.377769e-02,
                    -2.771053e-03, 8.241710e-04, -9.999958e-01, -5.542117e-02,
                    9.999644e-01, 7.969825e-03, -2.764397e-03, -2.918589e-01]
  R0_rectV:        [ 9.999454e-01, 7.259129e-03, -7.519551e-03, 
                   -7.292213e-03, 9.999638e-01, -4.381729e-03,
                    7.487471e-03, 4.436324e-03, 9.999621e-01]
  P0V:             [ 7.188560e+02, 0.000000e+00, 6.071928e+02, 0.000000e+00,
                    0.000000e+00, 7.188560e+02, 1.852157e+02, 0.000000e+00,
                    0.000000e+00, 0.000000e+00, 1.000000e+00, 0.000000e+00]

  #clustering
  leaf: 1
  object_z_axis_min: -50.0
  object_z_axis_max: 50.0
  object_x_axis_min: -50.0
  object_x_axis_max: 50.0
  object_y_axis_min: -50.0
  object_y_axis_max: 50.0
  object_radius_min: 5.0
  object_radius_max: 50.0
  object_k_merging_threshold_: 5.0
  cluster_size_min: 20